res/values/strings.xml

```diff
@@ -530,6 +530,18 @@
     <string name="systemmsg_action_by_me">%1$s by me.</string>
     <!-- Translators: %1$s will be replaced by sth. as "member xy added" (trailing full-stop removed). %2$s will be replaced by the name/addr of the person who did this action. -->
     <string name="systemmsg_action_by_user">%1$s by %2$s.</string>
+    <string name="systemmsg_unknown_sender_for_chat">Unknown Sender for this chat. See \'info\' for more details.</string>
```

Comment:
I think it should be "Unknown ***s***ender", and also fixed in the core.

Reply:
https://github.com/deltachat/deltachat-core-rust/commit/d59475f9bbab2f7180f957f9c462947bc01b26c6

Reply:
fixed